awx/api/views/analytics.py

```diff
@@ -0,0 +1,274 @@
+import requests
+import logging
+import urllib.parse as urlparse
+
+from django.conf import settings
+from django.utils.translation import gettext_lazy as _
+from django.utils import translation
+
+from awx.api.generics import APIView, Response
+from awx.api.permissions import IsSystemAdminOrAuditor
+from awx.api.versioning import reverse
+from awx.main.utils import get_awx_version
+from rest_framework.permissions import AllowAny
+from rest_framework import status
+
+from collections import OrderedDict
+
+AUTOMATION_ANALYTICS_API_URL_PATH = "/api/tower-analytics/v1"
+
+CERT_PATH = "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+
+ERROR_UPLOAD_NOT_ENABLED = "analytics-upload-not-enabled"
+ERROR_MISSING_URL = "missing-url"
+ERROR_MISSING_USER = "missing-user"
+ERROR_MISSING_PASSWORD = "missing-password"
+ERROR_NO_DATA_OR_ENTITLEMENT = "no-data-or-entitlement"
+ERROR_NOT_FOUND = "not-found"
+ERROR_UNAUTHORIZED = "unauthorized"
+ERROR_UNKNOWN = "unknown"
+ERROR_UNSUPPORTED_METHOD = "unsupported-method"
+
+logger = logging.getLogger('awx.api.views')
+
+
+class MissingSettings(Exception):
+    """Settings are not correct Exception"""
+
+    pass
+
+
+class GetNotAllowedMixin(object):
+    def get(self, request, format=None):
+        return Response(status=status.HTTP_405_METHOD_NOT_ALLOWED)
+
+
+class AnalyticsRootView(APIView):
+    permission_classes = (AllowAny,)
+    name = _('Automation Analytics')
+    swagger_topic = 'Automation Analytics'
+
+    def get(self, request, format=None):
+        data = OrderedDict()
+        data['reports'] = reverse('analytics_reports_list')
```

Comment:
this url is quite unclear for me. the `/api/v2/` is clear, but there are almost all endpoints POST only. 
List of reports can't be shown, because it doesn't go from database, but from c.th.c. Analytics API.
Is it a *must* to have it, or can I just remove the `/api/v2/analytics/` endpoint at all?

Reply:
I removed the analytics root endpoint

Reply:
added back by john's commit